src/illumidesk/handlers/lms_grades.py

```diff
@@ -0,0 +1,282 @@
+import os
+import json
+import time
+import logging
+import sys
+from pathlib import Path
+from tornado import web
+from xml.etree import ElementTree as etree
+from jupyterhub.handlers import BaseHandler
+from nbgrader.api import Gradebook
+from nbgrader.api import MissingEntry
+from pylti.common import post_message
+from filelock import FileLock
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+class GradesSenderError(Exception):
```

Comment:
Let's add docstrings here, I know its obvious what the exceptions are for but it would help us publish the docs later without rework.

Reply:
I agree